src/System.Diagnostics.Tracing/tests/BasicEventSourceTest/Harness/EventTestHarness.cs

```diff
@@ -0,0 +1,186 @@
+﻿using System.Diagnostics.Tracing;
+#if USE_ETW
+using Microsoft.Diagnostics.Tracing.Session;
+#endif
+using Xunit;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Threading;
+
+//Our tests shouldn't be parallelized. They often use the same EventSource, and only one instance can be active at once.
+[assembly: CollectionBehavior(DisableTestParallelization = true)]
```

Comment:
Other libraries in corefx keep this attribute in a separate file
https://github.com/dotnet/corefx/search?p=2&q=CollectionBehavior&utf8=%E2%9C%93